### PR TITLE
Mock UG (Preset Tasks & Modules)

### LINKED ISSUE #33: Update Mock UG
Create Mock UG for Preset Task and Module Commands.

### PR DESCRIPTION
**Pull request by yanlynnnnn**
Create Mock UG for Preset Task & Module commands

### CODE REVIEW

**Review by khoongwk — APPROVED** [review]
Nice, lots of commands added to UG. LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -66,6 +66,118 @@ Shows a message explaning how to access the help page.
...
+### Create a Preset Task Template : `maketemplate`
+
+Create a template for recurring weekly tasks (Preset Tasks). It is initialized
+with no Preset Tasks.
+
+Format : `maketemplate n/TEMPLATENAME`
+
+### Add a Preset Task to a Template : `addpresettask` 
+
+Add a Preset Task to a Task Template, specified by its index.
```
Comment (RuiFengg) [inline docs/UserGuide.md:90]: TaskTemplate instead of Task Template/Preset Task Template

File: docs/UserGuide.md [hunk 2]
```diff
@@ -66,6 +66,118 @@ Shows a message explaning how to access the help page.
...
+*  `deletetask i/1`
+
+### Create a Preset Task Template : `maketemplate`
+
+Create a template for recurring weekly tasks (Preset Tasks). It is initialized
+with no Preset Tasks.
+
+Format : `maketemplate n/TEMPLATENAME`
+
+### Add a Preset Task to a Template : `addpresettask`
```
Comment (RuiFengg) [inline docs/UserGuide.md:88]: Should Preset Task just be Task only?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -66,6 +66,118 @@ Shows a message explaning how to access the help page.
...
+
+Format :  `listpresettasks i/INDEX`
+- The index must be a **positive integer** and must be **valid**
+
+Example
+* `listpresettasks i/1`
+
+### Load Template into Task Schedule : `loadtemplate` 
+
+Load a Preset Task Template into the Task Schedule, specified by its index.
```
Comment (RuiFengg) [inline docs/UserGuide.md:120]: For now maybe we should use remove the spacing between Task Template, Task Schedule, Module Schedule etc.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -66,6 +66,118 @@ Shows a message explaning how to access the help page.
...
+
+- New Module Tasks must not clash with existing Module Tasks in the Module Schedule
+- The index of the Preset Module is its index in the Preset Module List 
+
+Example: 
+* `loadmodule i/100`
+
+### Remove a Module : `removemodule`
+
+Remove a Module and its Module Tasks from the Module Schedule, specified by its index.
```
Comment (RuiFengg) [inline docs/UserGuide.md:174]: Probably should try to use module code instead of index

**Comment by khoongwk** [comment]
Resolved Merge Conflict in User Guide. Needs further edits to UG, but let's merge first @yanlynnnnn got issues.

**Pull request merged**